pysat/_instrument.py

```diff
@@ -73,10 +73,11 @@ class Instrument(object):
     file_format : str or NoneType
         File naming structure in string format.  Variables such as `year`,
         `month`, `day`, etc. will be filled in as needed using python
-        string formatting.  The default file format structure is supplied in the
-        instrument `list_files` routine. See
+        string formatting.  The default file format structure is supplied
+        in the instrument `list_files` routine. See
         `pysat.files.parse_delimited_filenames` and
         `pysat.files.parse_fixed_width_filenames` for more information.
+        If unspecified, will be None.
```

Comment:
```suggestion
        The value will be None If not specified by the user at instantiation.
```

I believe this used to be a hidden parameter hence the lack of assignment for developer provided values.